charts/generic-govuk-app/values.yaml

```diff
@@ -14,12 +14,33 @@ replicaCount: 2
 podDisruptionBudget: {}
 
 # workerEnabled should be true if the app uses Sidekiq, false otherwise.
-workerEnabled: false
+workerEnabled: true
```

Comment:
How come we're changing this default? Do we need to update the app values where using the default?

Reply:
Oh my bad, I changed it during testing and forgot to change it back